perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -293,9 +326,145 @@ def _MakeSwiftCommandPrefix(auth_url, tenant_name, username, password):
   return ' '.join(options)
 
 
+def _ParseMultiStreamResults(raw_result, metadata=None):
+  """Read results from the api_multistream worker process.
+
+  Args:
+    raw_result: string. The stdout of the worker process.
+    metadata: dict. Base sample metadata
+
+  Returns:
+    A list of sample.Sample objects.
+  """
+
+  if metadata is None:
+    metadata = {}
+
+  records = json.loads(raw_result)
+  results = []
+
+  for record in records:
+    metric = 'Multi-stream %s KB %s' % (record['size'], record['operation'])
+    md = metadata.copy()
+    md['stream_num'] = record['stream_num']
+    md['object_size_b'] = record['size']
+
+    results.append(sample.Sample(metric,
+                                 record['latency'],
+                                 'sec',
+                                 md,
+                                 record['start_time']))
+
+  return results
+
+
+def _StringToBytes(string):
+  """Convert an object size, represented as a string, to bytes.
+
+  Args:
+    string: the object size, as a string with a quantity and a unit.
+
+  Returns:
+    an integer. The number of bytes in the size.
```

Comment:
According to my experiements, Python ints should hold sizes up to 2^63 bytes = 8 exabytes, so I think this is fine. I did originally support longs, and I only removed that support when I ran into an issue with printing them out and reading them back in, because of the "L" on the end of their printed representation. I can fix that, but I think that ints work for large enough sizes that it shouldn't be an urgent issue.